Source/ActivityIndicatorView.swift

```diff
@@ -281,16 +281,21 @@ public struct ActivityIndicatorView: View {
     }
 
     func createGradientIndicator(colors: [Color]) -> AnyView {
-        let colors = Gradient(colors: colors)
-        let conic = AngularGradient(gradient: colors, center: .center, startAngle: .zero, endAngle: .degrees(360))
+        let gradientColors = Gradient(colors: colors)
+        let conic = AngularGradient(gradient: gradientColors, center: .center, startAngle: .zero, endAngle: .degrees(360))
 
         let indicator = GeometryReader { (geometry: GeometryProxy) in
-            Circle()
-                .strokeBorder(conic, lineWidth: 4)
-                .frame(width: geometry.size.width, height: geometry.size.height)
-                .rotationEffect(!self.isAnimating ? .degrees(0) : .degrees(360))
-                .animation(Animation.linear(duration: 1.5)
-                    .repeatForever(autoreverses: false))
+            ZStack {
+                Circle()
+                    .stroke(colors.first ?? .white, lineWidth: 4)
+                Circle()
+                    .trim(from: 0.01, to: 0.99)
```

Comment:
That relies on stroke width of exactly 4, doesn't it? Would it work if we were to add stroke width as a parameter (which we might and maybe should)?

Reply:
Fixed, should work for different lineWidth within reasonable limits. 
Should it add as a parameter? I think if we will add it, we have to add this feature to other indicators.

Reply:
Different indicators might need lots of different customisation parameters - not only width, but I think we'll add them once somebody needs them, so let's leave it at that for now